app/src/main/java/com/example/benchproject/app/movie/details/viewModel/MovieDetailsMapper.kt

```diff
@@ -0,0 +1,23 @@
+package com.example.benchproject.app.movie.details.viewModel
+
+import com.example.benchproject.app.movie.details.entity.Genres
+import com.example.benchproject.app.movie.details.entity.MovieDetails
+import com.example.benchproject.domain.movie.details.entity.MovieDetailsModel
+import javax.inject.Inject
+
+class MovieDetailsMapper @Inject constructor() {
```

Comment:
No, there is no need to create classes for such a thing, it is totally useless and adds a lot of redundant classes. For such cases, you can use `object` or just function in the file (in Kotlin you don't need to create a class for everything like in Java). Second thing is that mapper could be just placed in the corresponding model file/class.

Reply:
Ok, I understand and I deleted it